client/payment-details/summary/index.tsx

```diff
@@ -367,18 +368,47 @@ const PaymentDetailsSummary: React.FC< PaymentDetailsSummaryProps > = ( {
 						<CardFooter className="payment-details-capture-notice">
 							<div className="payment-details-capture-notice__section">
 								<div className="payment-details-capture-notice__text">
-									{ `${ __(
-										'You need to capture this charge before',
-										'woocommerce-payments'
-									) } ` }
-									<b>
-										{ dateI18n(
+									{ createInterpolateElement(
+										__(
+											'You need to <a>capture</a> this charge in ',
+											'woocommerce-payments'
+										),
+										{
+											a: (
+												// eslint-disable-next-line jsx-a11y/anchor-has-content, react/jsx-no-target-blank
+												<a
+													href="https://woocommerce.com/document/woocommerce-payments/settings-guide/authorize-and-capture/#capturing-authorized-orders"
+													target="_blank"
+													rel="noreferer"
+												/>
+											),
+										}
+									) }
+									<abbr
+										title={ dateI18n(
 											'M j, Y / g:iA',
 											moment
 												.utc( authorization.created )
 												.add( 7, 'days' )
 										) }
-									</b>
+									>
+										<b>
+											{ sprintf(
+												// eslint-disable-next-line @wordpress/i18n-no-placeholders-only
+												__(
+													'%s',
+													'woocommerce-payments'
+												),
+												// Wrapped in __() to ensure translation of relative time strings inserted by moment.fromNow() (e.g. "days", "hours", "minutes", etc.)
```

Comment:
I'm afraid that this will not work because `moment.fromNow()` is dynamic, so translators will only see that we want to translate '%s', but not the actual time left. A tip in case you did not know, If you run `npm run build:client` you can have a look at the .pot file that translators will use. I believe this is what it shows for this PR:

```
#: dist/index.js:1
#: client/payment-details/summary/index.tsx:192
msgid "%s"
msgstr ""
```

Instead, I can think of a couple of options:
1. We would have more than one sentence for translators, depending on how close we are to the deadline. If it is days something like ` __( 'You need to capture this charge in %s days' ), $timeToCapture )` and if it is hours ` __( 'You need to capture this charge in %s hours' ), $timeToCapture )`. I believe you get the point.
2. Maybe moment.js is capable of localizing that string on its own. I believe we are able to know the locale of the user. If something like this is possible, it will be less verbose than the other option.

Or maybe there's another, more clever way of doing this. I'll also share this [i18n Best Practices](https://fieldguide.automattic.com/i18n/developers/best-practices/) link in case it is useful.

Reply:
@mgascam . Thanks for the awesome pointers and help! 

I have managed to make changes that use Moment's i18N feature.  Two questions / concerns I wanted to check with you and others. 

1.  The [total package size](https://github.com/Automattic/woocommerce-payments/pull/5897#issuecomment-1488653592) seems to increase by 7% ( 73KB ) . Would that be fine or unacceptable? If not, I am happy to work via alternative method of using individual strings ( although verbose ) 

2. I am unable to find a solution to the typescript error I get for including the `moment-with-locales` [component here ](https://github.com/Automattic/woocommerce-payments/pull/5897/files#diff-a41f2e2fca1befa933534f9c72b4d1552d19af998a9dc2252f592d31e10b7a59R10) . I had to add an `ts error expected` exclusion note for now. If I could get some pointers or help. 

Thanks so much

Reply:
> I have managed to make changes that use Moment's i18N feature. Two questions / concerns I wanted to check with you and others.
> 
>     1. The [total package size](https://github.com/Automattic/woocommerce-payments/pull/5897#issuecomment-1488653592) seems to increase by 7% ( 73KB ) . Would that be fine or unacceptable? If not, I am happy to work via alternative method of using individual strings ( although verbose )

I'm glad that you worked your way through it, congrats! But If you look at the size of the `index.js`, it grows  +72.6 kB (+32%). I'd say that's a bit too much, specially if we are not making a heavy use of moment's locales in other places.  I'd rather go the direction of having individual strings in this case.

>     2. I am unable to find a solution to the typescript error I get for including the `moment-with-locales` [component here ](https://github.com/Automattic/woocommerce-payments/pull/5897/files#diff-a41f2e2fca1befa933534f9c72b4d1552d19af998a9dc2252f592d31e10b7a59R10) . I had to add an `ts error expected` exclusion note for now. If I could get some pointers or help.

Perhaps not relevant after my previous comment, but maybe you are getting an error because we are missing the type definitions of the moment library. As you may know, this type definitions can be often found on [github](https://github.com/aquavit/DefinitelyTyped/blob/master/moment/moment.d.ts) and added to your project (and those are not added to the bundle, in case you wondered)

Reply:
Hi @mgascam . When you get time can you please review my latest changes? 

1. I have removed MomentJS and its locale, and tried implementing using plain vanilla. It seems to work fine in manual tests. 

2. On the test snapshots however something strange is happening, where you may kindly guide me. The current date and time is mocked at `Jan 1, 2023, 01:00 AM` .. The snapshots however seem to render `Jan 7, 2023, 3:00PM` , instead of what I would expect `Jan 8, 2023, 01:00AM`.  I may be doing some silly mistake there. For now to make the tests pass, I have matched the expected string to what the snapshot ends up throwing. Would appreciate your review and guidance 🙌

Reply:
Thanks for your guidance, @mgascam ! It now seems to be working fine!